decidim-meetings/app/assets/javascripts/decidim/meetings/meetings_filters.js.es6

```diff
@@ -0,0 +1,46 @@
+// = require_self
+$(document).ready(() => {
```

Comment:
We should bind this event on `turbolinks:load` as well. I recommend moving this code into a class, something like `MeetingFilterFormComponent`. Then you can bind both events `document:ready` and `turbolinks:load` and create an instance of this class (or method, or whatev) to bind the form events.
Also I'm not a huge fan of exposing random globals so it's better if you use a namespace to include the previous class (or method, or whatev) into a global called `DecidimMeetings`.